src/PushCommand.php

```diff
@@ -1,7 +1,18 @@
 <?php
+namespace Elendev\NexusComposerPush;
 
+if (file_exists(dirname(__DIR__) . '/vendor/autoload.php')) {
+    $loader = require_once dirname(__DIR__) . '/vendor/autoload.php';
+} elseif (file_exists(dirname(__DIR__) . '/../../autoload.php')) {
+    $loader = require_once dirname(__DIR__) . '/../../autoload.php';
+} else {
+    // error?
```

Comment:
I propose that a notice/debug is written in the console here, something like
```suggestion
  $this->getIO()
      ->write(
          '[GuzzleLoad] Failed to find Guzzle in vendors',
          true,
          IOInterface::VERY_VERBOSE
      )
    ;
```